### ISSUE TITLE
Add additional event changes to history 

### ISSUE THREAD

**Issue by Quinn-D** [neinteractiveliterature/intercode#8057]
It would be helpful to have the event history log, and change notification emails include more items.  Specifically the event GMs/hosts, and sign up bucket numbers.  Suggest reviewing if there are any other fields that should be covered in order to be monitored.

**Comment by nbudin** [comment]
Makes sense, thanks Quinn!  I'm actually surprised bucket sizes aren't showing up in the change list, since those are theoretically supposed to be logged, so I think that's a bug.  Event team member changes will be a little harder to implement, so I'm thinking I might tackle bucket sizes first.  Stay tuned!

**Comment by Quinn-D** [comment]
I thought bucket sizes were as well, but I didn't get notification when Kevin updated Beginning
